awx/main/tasks.py

```diff
@@ -274,6 +277,12 @@ def handle_setting_changes(setting_keys):
     logger.debug('cache delete_many(%r)', cache_keys)
     cache.delete_many(cache_keys)
 
+    if any([
+        setting.startswith('LOG_AGGREGATOR')
```

Comment:
What about settings that begin with `LOGGING` ? e.g: `LOGGING` and `LOGGING_SOCKET`? Should that also trigger a reconfiguration?

Reply:
Changes to `LOGGING` and `LOGGING_SOCKET` aren't managed by the API and require service restarts.